docs/application/native/guides/app-management/app-controls.md

```diff
@@ -808,6 +812,49 @@ The following common application controls are available:
 - [Voice Recorder](common-appcontrols.md#voice-recorder)
 - [VPN Service](common-appcontrols.md#vpn-service)
 
+<a name="URI"></a>
+## Application control uniform resource identifier (URI)
+
+> [!Note]
+>
+> Application Control URI is supported since Tizen 5.5.
```

Comment:
```suggestion 
> [!NOTE]
> Application Control URI is supported since Tizen 5.5.
```

NOTE is in capital letters 
Also, we dont need line space between [!NOTE] and the text.